orchestra/todos/auth.py

```diff
@@ -34,29 +33,17 @@ class IsAssociatedWithProject(permissions.BasePermission):
 
     def has_permission(self, request, view):
         worker = Worker.objects.get(user=request.user)
```

Comment:
i liked in the old format that it was explicit that GET and POST got their arguments from different sources. what are some of the benefits of this new implicit approach?

Reply:
We now have PATCH and PUT requests. Besides, `query_params` is not exclusive to GET requests anymore –– our filtering logic is based on it and can potentially be used across all request types :)

Reply:
ok cool -> want to just explain why we look for project_id in so many places? :)